packages/toast/src/toast.provider.tsx

```diff
@@ -90,6 +91,18 @@ export type ToastProviderProps = React.PropsWithChildren<{
   portalProps?: Pick<PortalProps, "appendToParentPortal" | "containerRef">
 }>
 
+export interface ToastDataContext {
+  topLevelDefaultOptions: UseToastOptions | undefined
```

Comment:
We could use "defaultOptions" or "globalOptions"

```suggestion
defaultOptions: UseToastOptions | undefined
```